### PR TITLE
Feat: Landmark Detection Model

### PR DESCRIPTION
**Pull request by yashksaini-coder**
# Pull Request for PyVerse 💡

## Issue Title : 📃: Landmark Detection Web Application

- **Info about the related issue (Aim of the project)** : 
- **Name:** Yash K. Saini
- **GitHub ID:** [
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a WoB 2024 participant it's, `WoB Participant`)** 

Closes: #114 

### Describe the add-ons or changes you've made 📃

This pull request introduces several important changes to the `Machine_Learning/Landmark_Detection` project. The changes include adding a `.gitignore` file, creating a `CONTRIBUTING.md` guide, including an `LICENSE` file, updating the `README.md`, and implementing a basic Streamlit app for landmark detection.

### Project Setup and Documentation:

* **Added `.gitignore` file**: This file includes common patterns to ignore Python-related files and directories, ensuring a cleaner repository. (`Machine_Learning/Landmark_Detection/.gitignore`)
* **Created `CONTRIBUTING.md`**: This guide provides instructions on how to report bugs, suggest features, and submit pull requests, encouraging community contributions. (`Machine_Learning/Landmark_Detection/CONTRIBUTING.md`)
* **Included `LICENSE` file**: Added the Apache License 2.0 to define the terms and conditions for using, reproducing, and distributing the project. (`Machine_Learning/Landmark_Detection/LICENSE`)
* **Updated `README.md`**: Briefly describes the purpose of the Landmark Detection project. (`Machine_Learning/Landmark_Detection/README.md`)

### Application Implementation:

* **Implemented basic Streamlit app**: The app allows users to upload an image and classify it using a pre-trained TensorFlow model. (`Machine_Learning/Landmark_Detection/app.py`)

## Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, local variables)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## How Has This Been Tested? ⚙️

Describe how it has been tested
Describe how have you verified the changes made

## Checklist: ☑️

- [X] My code follows the guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.
- [X] Any dependent changes have been merged and published in downstream modules.

********************************************************************************************

@UTSAVS26 review this PR & add labels to it. `gssoc-ext`, `hacktoberfest`, `hacktoberfest-accpeted`, `Level 3`

### CODE REVIEW

**Comment by UTSAVS26** [comment]
@yashksaini-coder remove unnecessary files like `contributing` and 'licence` and `.gitignore` they are not required.

**Comment by yashksaini-coder** [comment]
> unnecessary files

@UTSAVS26 All files removed and the changes are merged. check it now
